src/interfaces/wallet.h

```diff
@@ -247,6 +247,9 @@ class Wallet
     //! Register handler for watchonly changed messages.
     using WatchOnlyChangedFn = std::function<void(bool have_watch_only)>;
     virtual std::unique_ptr<Handler> handleWatchOnlyChanged(WatchOnlyChangedFn fn) = 0;
+
+    //! Abort a rescan
```

Comment:
Nit, period.

Reply:
Would suggest renaming this method `abortRescan` and moving it above `backupWallet` (both here and in `wallet.cpp`). These interfaces are organized with regular methods on top and callback methods below. Also roughly with whole-wallet operations above operations on individual addresses and transactions.

Reply:
Done (moved and added period).